src/org/opensolaris/opengrok/authorization/AuthorizationFramework.java

```diff
@@ -160,6 +160,12 @@ public boolean isAllowed(HttpServletRequest request, Project project) {
             public boolean decision(IAuthorizationPlugin plugin) {
                 return plugin.isAllowed(request, project);
             }
+
+            @Override
```

Comment:
I changed the indentation however this was the default NB format feature so maybe in the future it will again shift to the left